.github/workflows/pr_build.yml

```diff
@@ -63,11 +63,15 @@ jobs:
           cache: 'maven'
       - name: Build with Maven
         run: mvn -B install --file pom.xml
-      - name: Build Gradle Example
+      - name: Build Gradle Example - Java
         if: ${{ matrix.java == '8' }} # Gradle example can only be built on Java 8
         run: |
           cd examples/powertools-examples-core/gradle
           ./gradlew build
+      - name: Build Gradle Example - Kotlin
+        run: |
```

Comment:
should we build it only with a specific version ? like java ?

Reply:
I don't see a reason to limit this, it should work with any JDK version

Reply:
The gradle one we limit because there the wrapper and its aspect library can't work for both java8 and java11+ - I don't believe that affects kotlin 🙏